test/core/end2end/tests/cancel_in_a_vacuum.cc

```diff
@@ -93,7 +93,7 @@ static void test_cancel_in_a_vacuum(grpc_end2end_test_config config,
   grpc_call* c;
   grpc_end2end_test_fixture f =
       begin_test(config, "test_cancel_in_a_vacuum", mode, nullptr, nullptr);
-  cq_verifier* v_client = cq_verifier_create(f.cq);
+  grpc_core::CqVerifier v_client(f.cq);
```

Comment:
Doesn't look like this is actually used here, so it can probably just be removed.

Reply:
done